packages/react/src/components/gdpr-consent-dialog/consent-dialog.tsx

```diff
@@ -68,6 +69,25 @@ export const ConsentDialog = ({
     }
   }, [open]);
 
+  const TriggerButton = forwardRef<HTMLButtonElement>((props, ref) => (
```

Comment:
Wouldn't it be best to pull this out of the `ConsentDialog` component?

Reply:
I think as a default probably no 🤔 

If this is its own component it increases the burden of complexity for the minimum implementation

Reply:
Done, is it better now?